front/app/components/admin/ActionForm/DataCollection/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { Box, Text, Radio, Title } from '@citizenlab/cl2-component-library';
+
+import { PermittedBy } from 'api/phase_permissions/types';
+
+interface Props {
+  allow_anonymous_participation: boolean;
+  permitted_by: PermittedBy;
+  onChange: () => void;
+}
+
+const DataCollection = ({
+  allow_anonymous_participation,
+  permitted_by,
+  onChange,
+}: Props) => {
+  return (
+    <Box mt="32px">
+      <Title variant="h4" color="primary">
+        User data collection
```

Comment:
Going to do this copy the proper way if everyone is fine with it in the epic review